src/app.py

```diff
@@ -0,0 +1,98 @@
+import praw
+import re
+from datetime import datetime
+import time
+import algorithm
+import constants
+import cache
+
+def main(thresholds: constants.Thresholds):
+    reddit = praw.Reddit('MemeAdviser')
+
+    replied_file = cache.AbstractFile("../data/replied.txt")
+    subscribed_file = cache.AbstractFile("../data/subscribed.txt")
+    fileLoadCache = cache.FileCache()
+    replied = fileLoadCache.refresh(replied_file)
+    subscribed = fileLoadCache.refresh(subscribed_file)
+    fileLoadCache.add(replied)
+    fileLoadCache.add(subscribed)
+
+    subreddit = reddit.subreddit("MemeEconomy")
+    post_subreddit = reddit.subreddit("InsiderMemeTrading")
+    submissions = subreddit.hot()
+    submission = next(submissions)
+
+    while submission.stickied:
+        submission = submissions.next()
+
+    # Find out how old the post is
+    minutes = int(round((time.time() - submission.created_utc) / 60))
+
+    # Store time in hours
+    if minutes >= 60:
+        minutes = str(minutes // 60) + "h " + str(minutes % 60) + "min"
+    else:
+        minutes = str(round((time.time() - submission.created_utc) / 60)) + " minutes"
+
+    if submission.id not in replied:
+        try:
+            # Update replied.txt
+            replied.append(submission.id)
+            for post_id in replied:
+                replied.write_to_file(post_id + "\n")
```

Comment:
```
Traceback (most recent call last):
  File "app.py", line 97, in <module>
    main(constants.Thresholds)
  File "app.py", line 42, in main
    replied.write_to_file(post_id + "\n")
AttributeError: 'list' object has no attribute 'write_to_file'
```

Reply:
Something is happening with a variable conflict

Reply:
I can't seem to find any conflicts... `replied` is only declared once, on line 15
It seems that `write_to_file` is a method of the `FileCache` class, but 
```python
replied = fileLoadCache.refresh(replied_file)
``` 

returns a list

Reply:
Ah, that’s it.  Any idea how to work around this?

Reply:
I don't really understand `cache.py` but we could replace '`replied.write_to_file()` with 

```python 
with open("../replied.txt", "w") as f: 
    for post_id in replied: 
        f.write(post_id + "\n")
```

like it was before... Or does that just make the whole thing pointless?

Reply:
I need to take a break from this PR for my mental health, I’ll reopen it once I have a fix.

Reply:
Take your time, don't worry about finishing it if you don't want to